### PR TITLE
Create HTML page with two search bars

### PR DESCRIPTION
**Pull request by ychaiun**
- [DEMO LINK](https://ychaiun.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://ychaiun.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,33 @@
...
+      action="#"
+      method="post"
+      data-qa="big"
+      class="big-form"
+    >
+      <input
+        type="search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-field search-field--big icon icon--big"
```
Comment (Anastasiia-Svintsova) [inline src/index.html:25]: Why do you need such a lot of classes here? It's not an icon, as I see
```suggestion
        class="search-field search-field--big"
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- Please, make sure you deploy the right page 
image
Check [this link](https://stackoverflow.com/questions/48919200/github-pages-only-showing-readme-file) to fix the issue. If you have any questions, ask them in the fe_chat

- Also fix a test report link, please 
image

File: src/index.html [hunk 2]
```diff
@@ -10,7 +10,33 @@
...
+      >
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
```
[the remark this answers is not in the document]
Comment (witflash) [reply src/index.html:32]: @ychaiun 
Don't leave unresolved comments, please

**Review by witflash — APPROVED** [review]
Good job

**Pull request closed without merging**